kimchi/src/linearization.rs

```diff
@@ -117,6 +121,11 @@ pub fn linearization_columns<F: FftField + SquareRootField>(
         h.insert(Witness(i));
     }
 
+    // the coefficient polynomials
+    for i in 0..COLUMNS {
```

Comment:
Cool.  Part of this PR?

Reply:
No idea why this is marked as modified, I did not introduce that, it is just part of the coefficient PR